.github/workflows/model_registry_test.yaml

```diff
@@ -80,3 +88,9 @@ jobs:
         echo "Dry-run KF Model Registry REST API..."
         export KF_TOKEN="$(kubectl -n default create token default)"
```

Comment:
Thank you for the PR. All model registry pods should live in the Kubeflow namespace. and AFAIK only default-editor from the example user namespace should be able to reach it. Everything else is probably a security breach.

Reply:
> Thank you for the PR. All model registry pods should live in the Kubeflow namespace. and AFAIK only default-editor from the example user namespace should be able to reach it. Everything else is probably a security breach.

Circling back to this conversation, we brought it yesterday in the MR community, are we supossed to change this, @tarilabs and the rest of the team were confused since this has been working just fine as it was, is it requried by us to change both tests?